PrivateCollections/Ethiopia/MentasnotAbbuhay/MEAG001.xml

```diff
@@ -0,0 +1,497 @@
+                              </item>
+                              <item xml:id="q16" n="16">
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="149r" to="158v"/>
+                              </item>
+                              <item xml:id="q17" n="17">
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="159r" to="168v"/>
+                              </item>
+                              <item xml:id="q18" n="18">
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="169r" to="178v"/>
+                              </item>
+                              <item xml:id="q19" n="19">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="179r" to="186v"/>
+                              </item>
+                              <item xml:id="q20" n="20">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="187r" to="194v"/>
+                              </item>
+                              <item xml:id="q21" n="21">
+                                 <dim unit="leaf">4</dim>
+                                 <locus from="195r" to="198v"/>
+                              </item>
+                           </list>
+                        </collation>
+                        <condition key="good"/>
+                     </supportDesc>
+                  </objectDesc>
+                             
+                                               
+                  <bindingDesc>
+                     <binding contemporary="true" xml:id="binding">
+                        <decoNote xml:id="b1">Blind-tooled brown leather cover.</decoNote>
+                        <decoNote xml:id="b2" type="Boards">Wooden boards with multicoloured <term key="textileInlay">textile inlays</term></decoNote>
+                        <decoNote xml:id="b3" type="bindingMaterial">
+                           <material key="wood"/>
+                           <material key="leather"/>
+                           <material key="textile"/>
+                        </decoNote>
+                        <decoNote xml:id="b4" type="SlipCase"> The manuscript is kept in a māḫdar.</decoNote>
+                        <decoNote xml:id="b5" type="SewingStations">4</decoNote>
+                     </binding>    
+                  </bindingDesc>
+               </physDesc>
+               
+               <history>
+                  <origin>
+                     The manuscript can be dated based on the owner information to the
+                     <origDate notBefore="1900" notAfter="1950" evidence="context">
+                        early to mid twentieth century</origDate>.
+                     No colophon notes are present in the manuscript, but the current owner<persName ref="PPRS14050MentasnotAbbuhay"/>
```

Comment:
It might be useful to give a date for this information, because it maybe online also in a later time, when Mentasnot is not the owner anymore. However, I wonder how it can be written in the year 1900 by the father of the current owner (in 2024). This is 124 years after. How old is Mentasnot at the moment, that this is possible? Probably the time can be estimated more realistically to a time somewhat later e.g. 1940 to 1960?

Reply:
Thank you for updating the origDate. I think it is more realistic based on the information, that you gave! However, 1940 till 1970 is "mid twentieth century". The "late" twentieth century, I would say, starts not before 1980.